app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarIntegration.kt

```diff
@@ -114,11 +114,11 @@ class DefaultToolbarIntegration(
                 interactor.onTabCounterMenuItemTapped(it)
             },
             iconColor =
-                if (isPrivate) {
-                    ContextCompat.getColor(context, R.color.primary_text_private_theme)
-                } else {
-                    null
-                }
+            if (isPrivate) {
+                ContextCompat.getColor(context, R.color.primary_text_private_theme)
+            } else {
+                null
+            }
```

Comment:
Can we check the formatter to make sure this is desired?

Reply:
After rebasing I don't see the change

Reply:
Still seems like it's here?

Reply:
When you directly to files that have changed on the PR https://github.com/mozilla-mobile/fenix/pull/20536/files, I'm not able to see the change, are you?

Reply:
I moved the line above as ktlint was complaining